components/net/bluetooth_thread.rs

```diff
@@ -624,15 +623,11 @@ impl BluetoothManager {
             Some(a) => a,
             None => send_error!(sender, ADAPTER_ERROR),
         };
-        let mut value = match self.get_gatt_characteristic(&mut adapter, &id) {
-            Some(c) => Some(c.read_value().unwrap_or(vec!())),
-            None => None,
-        };
+        let mut value = self.get_gatt_characteristic(&mut adapter, &id)
+                            .map(|c| c.read_value().unwrap_or(vec![]));
```

Comment:
in the most cases we use vec!() what is the different between vec![] and vec!() ?